.github/workflows/tests-ipsec-upgrade.yaml

```diff
@@ -393,6 +393,13 @@ jobs:
             kubectl get pods --all-namespaces -o wide
             kubectl -n kube-system exec daemonset/cilium -c cilium-agent -- cilium status
 
+            # 1.12 is EOL, we have to run commands manually to clean up stale routing rules
+            cilium_pods=$(kubectl -nkube-system get po -l k8s-app=cilium --no-headers -o custom-columns=":metadata.name")
+            for cilium_pod in $cilium_pods; do
+              kubectl -nkube-system exec $cilium_pod -- ip -4 rule delete fwmark 0xB00/0xF00 lookup 2005 2>/dev/null || true
+              kubectl -nkube-system exec $cilium_pod -- ip -6 rule delete fwmark 0xB00/0xF00 lookup 2005 2>/dev/null || true
+            done
+
```

Comment:
This is something we only need for `[ "${{ matrix.mode }}" = "minor" ]`, right? For the `patch` test we already have everything what's needed in `v1.13`.

(if you agree, I think it's fine to fix that up in a follow-up PR. No need to re-run the whole CI for it)

Reply:
(I didn't receive email notification for this comment :fearful: )

Agree, thank you :+1: